src/middleware/validation.schema.js

```diff
@@ -217,6 +227,16 @@ export default [
     method: 'post',
     schema: loginSchema
   },
+  {
+    route: '/reset-password',
+    method: 'post',
+    schema: reset
+  },
+  {
+    route: '/auth/reset-password',
+    method: 'post',
+    schema: resetPassword
```

Comment:
Please can you add a schema at the end of this as `resetPassword ` doesn't give an idea of what the object does